src/lib/types/PVQuestion.d.ts

```diff
@@ -3,4 +3,5 @@ export interface PVQuestion {
 	questionId: string;
 	messageLink: string;
 	type: 'private' | 'anonymous';
+	threadId: string;
```

Comment:
The channel ID is already stored in the message link so this field is unneeded. Message links follow this format `https://discord.com/channels/{guildId}/{channelId/{messageId}`